src/components/input/range/index.tsx

```diff
@@ -32,8 +33,10 @@ export class SmoothlyInputRange {
 					type="range"
 					min={this.min}
 					max={this.max}
-					step={"any"}
-					onInput={e => e.target instanceof HTMLInputElement && (this.value = Number.parseFloat(e.target.value))}
+					step={this.step}
+					onInput={e =>
+						e.target instanceof HTMLInputElement && (this.value = Math.trunc(e.target.valueAsNumber * 100) / 100)
```

Comment:
I think we prefer to break this out as a method on the class `inputHandler(event: Event): void {...}` and use `onInput={e => this.inputHandler(e)}` in the JSX.